packages/help-center/src/components/help-center-feedback-form.tsx

```diff
@@ -1,19 +1,36 @@
 import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { getPlan } from '@automattic/calypso-products';
+import { HelpCenterSite } from '@automattic/data-stores';
+import { useGetOdieStorage } from '@automattic/odie-client';
 import { useI18n } from '@wordpress/react-i18n';
 import { addQueryArgs } from '@wordpress/url';
 import React, { useState } from 'react';
+import { useHelpCenterContext } from '../contexts/HelpCenterContext';
 import { ThumbsDownIcon, ThumbsUpIcon } from '../icons/thumbs';
+import HelpCenterContactSupportOption from './help-center-contact-support-option';
 import './help-center-feedback-form.scss';
 interface HelpCenterFeedbackFormProps {
 	postId: number;
 	blogId?: string | null;
 	slug?: string;
+	articleUrl?: string | null | undefined;
 }
-const HelpCenterFeedbackForm = ( { postId, blogId, slug }: HelpCenterFeedbackFormProps ) => {
+const HelpCenterFeedbackForm = ( {
+	postId,
+	blogId,
+	slug,
+	articleUrl,
+}: HelpCenterFeedbackFormProps ) => {
 	const { __ } = useI18n();
 	const [ startedFeedback, setStartedFeedback ] = useState< boolean | null >( null );
 	const [ answerValue, setAnswerValue ] = useState< number | null >( null );
 
+	const { sectionName, site } = useHelpCenterContext();
+	const wapuuChatId = useGetOdieStorage( 'chat_id' );
```

Comment:
Do we need this if it's not coming from Wapuu?

Reply:
Im not sure about this tbh. I'll merge as is for now and follow up.